CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ Start a discussion on the [repository issue tracker](https://github.com/dotnet/a
 ## Bugs and feature requests?
 **IMPORTANT: If you want to report a security related issue, please see the `Reporting security issues and bugs` section below.**
 
-Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.
+Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your unique scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.
```

Comment:
```suggestion
Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact helps us prioritize the most important issues to be worked on sooner rather than later.
```